lib/rtoolsHCK.rb

```diff
@@ -1211,6 +1233,23 @@ def install_machine_driver_package(machine,
     end
   end
 
+  # == Description
+  #
+  # Installs a driver certificate, (.cer file), on a machine.
```

Comment:
does it work with pfx as well?

Reply:
No.
The script that installs the certificate ignores the pfx file:
https://github.com/virtio-win/kvm-guest-drivers-windows/blob/master/Tools/InstallCertificate.bat
I assumed it was not essential.
I can add the support easily.